integration-tests/kafka-sasl-ssl/src/test/java/org/apache/camel/quarkus/kafka/sasl/KafkaSaslSslIT.java

```diff
@@ -16,10 +16,10 @@
  */
 package org.apache.camel.quarkus.kafka.sasl;
 
-import io.quarkus.test.junit.DisabledOnNativeImage;
 import io.quarkus.test.junit.NativeImageTest;
+import org.junit.jupiter.api.Disabled;
 
-@DisabledOnNativeImage("https://github.com/apache/camel-quarkus/issues/3239")
+@Disabled("https://github.com/apache/camel-quarkus/issues/2729")
```

Comment:
Need a new issue, that one was closed and fixed some time ago.

Reply:
yeah, I will raise a new issue to fix these native tests for ```KafkaSaslSslIT``` and ```KafkaSslTest```.

Reply:
and it should be fixed in main branch ?

Reply:
Ah you're right. It was already disabled on the `main` branch so I guess we should fix it there.

In that case we can probably merge this PR and take care of `KafkaSaslSslIT` on the `main` branch.

Reply:
yeah, I raise https://github.com/apache/camel-quarkus/issues/3254